packages/fluent-ui/src/DateTimeWidget/DateTimeWidget.tsx

```diff
@@ -77,7 +77,7 @@ const DateTimeWidget = ({
       disabled={disabled}
       readOnly={readonly}
       name={name}
-      type={schema.type as string}
+      type="datetime-local"
```

Comment:
Can you just render a `<TextWidget>` here?

Reply:
See https://github.com/epicfaace/react-jsonschema-form/blob/395d1726f15fede459bb21ee23d4c55331285dd8/packages/fluent-ui/src/TextareaWidget/TextareaWidget.tsx

Reply:
I did.